sentinel/src/config.js

```diff
@@ -38,7 +38,7 @@ const initFeed = () => {
   // Use since=now on ddoc listener so we don't replay an old change.
   const feed = new follow.Feed({ db: process.env.COUCH_URL, since: 'now' });
   feed.on('change', change => {
-    if (change.id === '_design/medic') {
+    if (change.id === 'settings') {
```

Comment:
is now the time to create a SETTINGS SHARED LIBRARY!?!?!?!?

Hyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyype

Reply:
Yeah maybe. I'll create a todo to look at if after this is merged.